app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transfers/TransfersViewModel.kt

```diff
@@ -49,6 +52,14 @@ class TransfersViewModel @Inject constructor(
 
     val selectedTransferUuids: SnapshotStateMap<String, Boolean> = mutableStateMapOf()
 
+    val isDeepLinkConsumed = savedStateHandle.getStateFlow(IS_DEEP_LINK_CONSUMED_KEY, false)
+
+    fun consumeDeepLink() {
+        viewModelScope.launch {
+            if (!isDeepLinkConsumed.first()) savedStateHandle[IS_DEEP_LINK_CONSUMED_KEY] = true
```

Comment:
Is that really necessary to check this ? `isDeepLinkConsumed.first()`.

If it's just to avoid settings the value in the `savedStateHandle` for optimization, maybe doing a `isDeepLinkConsumed.first()` is less optimized ? In my opinion, this optimization is not necessary.